Realm/Realm/Realm.csproj

```diff
@@ -116,6 +116,9 @@
   <ItemGroup>
     <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
   </ItemGroup>
+  <ItemGroup>
+	<Folder Include="Sync\ErrorHandling\" />
```

Comment:
Why do we need this?

Reply:
It's where I put the error handling for sync (all the client reset handlers). Instead of putting those in the sync folder. If you want I can put them in the parent sync folder instead.

Reply:
I mean, MSBuild should just pick up the folder, I don't think we need to explicitly add it in the project file.